sn_cmd_test_utilities/src/lib.rs

```diff
@@ -337,30 +326,19 @@ pub mod util {
         serde_json::from_str(output).expect("Failed to parse output of `safe keys create`")
     }
 
-    // Executes arbitrary `safe ` commands and returns
-    // output (stdout, stderr, exit code).
-    //
-    // If expect_exit_code is Some, then an Err is returned
-    // if value does not match process exit code.
+    /// Runs safe with the arguments specified, with the option to assert on the exit code.
+    /// This was changed to use the assert_cmd crate because the newer version of this crate
+    /// provides *both* the stdout and stderr if the process doesn't exit as expected. This is
+    /// extremely useful in this test suite because there are lots of commands used to setup the
+    /// context for the tests, and you need to be able to see why those fail too.
     pub fn safe_cmd(args: &[&str], expect_exit_code: Option<i32>) -> Result<process::Output> {
```

Comment:
Suggestion: Not too pressing since it was already like this, but this could be a bit more ergonomic with an `IntoIterator` (as with [`assert_cmd::Command::args`](https://docs.rs/assert_cmd/2.0.0/assert_cmd/cmd/struct.Command.html#method.args)):

```suggestion
    pub fn safe_cmd(args: impl IntoIterator<Item = &str>, expect_exit_code: Option<i32>) -> Result<process::Output> {
```

This would allow calling with with any iterable of stringy things. E.g. from elsewhere in the PR:

```rust
    let slice: Vec<&str> = args.iter().map(|x| &**x).collect();
    safe_cmd(&slice, expect_exit_code)
```

Could become just:

```rust
    safe_cmd(args, expect_exit_code)
```

Similarly, arrays now implement `IntoIterator`, so sites like:

```rust
safe_cmd_stdout(&["files", "put", TEST_FILE, "--json"], Some(0))
```

Could become:

```rust
safe_cmd_stdout(["files", "put", TEST_FILE, "--json"], Some(0))
```

Reply:
Changed this and it made all the calling code cleaner. Thanks.